cabal.project

```diff
@@ -14,9 +14,9 @@ repository cardano-haskell-packages
 -- update either of these.
 index-state:
   -- Bump this if you need newer packages from Hackage
-  , hackage.haskell.org 2023-11-20T23:52:53Z
+  , hackage.haskell.org 2023-12-16T00:00:00Z
   -- Bump this if you need newer packages from CHaP
-  , cardano-haskell-packages 2023-11-29T22:55:21Z
+  , cardano-haskell-packages 2023-12-16T00:00:00Z
```

Comment:
Until a new cabal is out, I prefer setting this to real values to avoid the annoying warning